### DISCUSSION TITLE
Node Red login details

### DISCUSSION THREAD

**Discussion by douvers** [tteck/Proxmox#1957] [Help Needed] [answered]
I've setup a node red lxc and I'm trying to sftp my packages.json file to it from an existing pi node red installation but I'm having difficulty logging in. I noticed that the default setup indicates Automatic Login but what are the login details? 

I setup another node red instance as a test and used the advanced setup and enabled root ssh access and specified a password & that allows me to sftp the file to it but I suppose its not a good idea to have root ssh access enabled. Have I missed something that indicates what the default user & password would be?  Thanks.

**Comment by tteck** [comment, marked as answer]
Automatic Login has no default password.

**Reply by tteck** [reply]
Undoing autologin is a pinned discussion

**Comment by rholze** [comment]
Hello, I installed the template for the PROXMOX node-red container on my PROXMOX server today. During the installation I also got an IP address displayed. Now I don't see an IP address under Summery. Calling the website <ip>:1880 does not work either. A user and a password are expected on the console, but I don't know them. An automatic logon is also not executed. How can I access the console to check the IP address and the status of the node-red service?
Many thanks Ronald
